source/features/clean-pinned-issues.css

```diff
@@ -1,10 +1,10 @@
 /* Changes the layout of pinned issues from side-by-side to a standard list. */
-.js-pinned-issues-reorder-container .f4 {
+.clean-pinned-issue .js-pinned-issues-reorder-container .f4 {
 	display: none !important; /* Hide title */
 }
 
 @media (min-width: 700px) {
-	:root .js-pinned-issues-reorder-list {
+	:root .clean-pinned-issue .js-pinned-issues-reorder-list {
```

Comment:
`:root` can likely be dropped now in both places. Can you make sure that it works?